### DISCUSSION TITLE
可以用子域名来映射本地的不同服务吗？

### DISCUSSION THREAD

**Discussion by Suchun-sv** [natfrp/wiki#532] [使用问题]
RT，尝试了本地反向代理路径，比如/redis -> localhost:6379, 但是类似的web网站就没发这么整，求求大佬指教，是没法用子域名吗？

**Comment by dezhishen** [comment]
你可以包一个反向代理，例如traefik
然后将隧道指向traefik的端口
traefik中配置 /aaa -> aaa:8080
